packages/core/src/domain/patient.ts

```diff
@@ -41,7 +41,7 @@ export type DriversLicense = {
   state: USState;
 };
 
-export const genderAtBirthTypes = ["F", "M"] as const;
+export const genderAtBirthTypes = ["F", "M", "UN", "UNK"] as const;
```

Comment:
`UN` and `UNK` are confusing, can we choose something else for `UN`?

Reply:
I mean we could but then we would have 3 different gender values:

fhir: male, female, other, unknown
IHE: M, F, UN, UNK
Metriport SDK: ???

Thats fine with me since this is customer facing and we want to keep things intuitive for customers, but it adds more complexity on our end since currently the types the SDK supports are the same the IHE gateways want.